vortexor/src/cli.rs

```diff
@@ -92,6 +92,19 @@ pub struct Cli {
     #[arg(long, value_parser = parse_port_range, value_name = "MIN_PORT-MAX_PORT", default_value = get_default_port_range())]
     pub dynamic_port_range: (u16, u16),
```

Comment:
How many dynamic ports does vortexor need? Maybe it could be easier to just make tpu_address and tpu_forwards_address required arguments and remove the dynamic_port_range argument?

Reply:
the dynamic_port_range is also used for client socket binds for rpc and websocket. We may add more sockets in the future such as for RPC service of the vortexor for subscription management.

Reply:
Oh I see, yes in this case it makes sense to have it to deconflict with agave. Also please make sure that defaults here are not the same as for agave itself, so if this is not provided we do not start accidentally binding on top of agave's default port range=)